packages/client/fluence-js/src/index.ts

```diff
@@ -17,21 +17,33 @@
 import log, { LogLevelDesc } from 'loglevel';
 export { KeyPair } from '@fluencelabs/keypair';
 
-import { FluencePeer, PeerConfig } from './internal/FluencePeer';
+import { FluencePeer, PeerConfig } from '@fluencelabs/js-peer/dist/FluencePeer';
+import { isBrowser, isNode } from 'browser-or-node';
 
-export { PeerStatus } from './internal/FluencePeer';
-export { FluencePeer, PeerConfig } from './internal/FluencePeer';
-export { MarineLoglevel as AvmLoglevel } from './internal/utils';
-export { PeerIdB58, CallParams } from './internal/commonTypes';
-export { loadWasmFromFileSystem, loadWasmFromNpmPackage, loadWasmFromServer } from '@fluencelabs/marine-js';
+export { PeerStatus } from '@fluencelabs/js-peer/dist/FluencePeer';
+export { FluencePeer, PeerConfig } from '@fluencelabs/js-peer/dist/FluencePeer';
+export { PeerIdB58, CallParams } from '@fluencelabs/js-peer/dist/commonTypes';
 
 export const setLogLevel = (level: LogLevelDesc) => {
     log.setLevel(level);
 };
 
 log.setDefaultLevel('WARN');
 
-const defaultPeer = new FluencePeer();
+let pkg: string;
```

Comment:
Could you please explain this package purpose in README. If it is temporary compatibility thing - I don't really understand how will it work with require in browser

Reply:
That was a temporary compatibility thing. It didn't work though, so I've changed this logic a little bit